service/tls.go

```diff
@@ -293,3 +300,10 @@ func defaultCipherSuitesMap() map[string]uint16 {
 	}
 	return ciphers
 }
+
+// warnIfRSAPrivateKey output warning log if the private key is RSA.
+func warnIfRSAPrivateKey(privateKey crypto.PrivateKey) {
+	if _, ok := privateKey.(*rsa.PrivateKey); ok {
+		glg.Warn("The private key is RSA. Consider using an ECDSA key for better performance.")
```

Comment:
```suggestion
		glg.Warn("The private key used in the server certificate is RSA. Consider using an ECDSA key for better performance.")
```
サーバー証明書の鍵の話であることを明記した方がよいかと思いました

Reply:
@y-myajima 修正しました。https://github.com/AthenZ/authorization-proxy/pull/43/commits/5c7e17cf02f6e8a6385fdfeb938f2d42245e188f